tests/src/test/scala/cats/tests/CokleisliTests.scala

```diff
@@ -19,6 +19,10 @@ class CokleisliTests extends SlowCatsSuite {
   def cokleisliEqE[F[_], A](implicit A: Arbitrary[F[A]], FA: Eq[A]): Eq[Cokleisli[F, A, A]] =
     Eq.by[Cokleisli[F, A, A], F[A] => A](_.run)
 
+  implicit val iso = MonoidalTests.Isomorphisms.covariant[Cokleisli[Option, Int, ?]]
```

Comment:
For all kinds that are different from `* −> *`, I had to add this line. I’d be happy to find a workaround.